cmd/shortener/main_test.go

```diff
@@ -223,3 +224,9 @@ func TestGzipCompression(t *testing.T) {
 		})
 	}
 }
+
+func Example() {
```

Comment:
Стоит вынести пример  в файл `example_test.go` и назвать пример в соответствии, что он демонстрирует.   
```suggestion
func ExampleSave() {
```

Reply:
Готово, перенес